README.md

```diff
@@ -1,268 +1,334 @@
+
+See [createIdentityResponseHandler](https://alphagov.github.io/passport-verify/modules/_create_identity_response_handler_.html#createidentityresponsehandler)
+and its [callbacks](https://alphagov.github.io/passport-verify/interfaces/_create_identity_response_handler_.responsescenarios.html) in the API documentation.
+
+<details>
+<summary>
+See an example of how to call the `createStrategy` method
+</summary>
+
+```javascript
+// route for authenticating a user
+app.post('/verify/start', passport.authenticate('verify'))
+
+// route for handling a callback from verify
+app.post('/verify/response', (req, res, next) => (
+
+   // in this example, authenticate() is being called from within the route handler
+   // rather than being used as middleware, this provides access to the request
+   // and response objects through closure
+   const authMiddleware = passport.authenticate('verify', function (error, identity, infoOrError, status) {
+
+    if (error) {
+      return res.send(`TODO: render error-page with message ${error: error.message}`)
+    }
+
+    if (identity) {
+      // passport-verify requires the use of a custom callback to handle successful
+      // authentication
+      return req.logIn(identity, () => res.send('TODO: redirect to service landing page')))
+    }
+
+    return res.send(`TODO: redirect to authentication failed page with ${error: infoOrError}`)
+
+  })
+  authMiddleware(req, res, next)
+```
+</details>
+
+If your service connects to GOV.UK Verify using a Matching Service Adapter, you should use the `createResponseHandler` method.
+
+See [createResponseHandler](https://alphagov.github.io/passport-verify/modules/_create_response_handler_.html#createresponsehandler)
+and its [callbacks](https://alphagov.github.io/passport-verify/interfaces/_create_response_handler_.responsescenarios.html#onauthnfailed) in the API documentation.
+
+<details>
+<summary>
+See an example of how to call the `createStrategy` method
+</summary>
+
+```javascript
+// route for authenticating a user
+app.post('/verify/start', passport.authenticate('verify'))
+
+// route for handling a callback from verify
+app.post('/verify/response', (req, res, next) => (
+
+ // in this example, authenticate() is being called from within the route handler
+ // rather than being used as middleware, this provides access to the request
+ // and response objects through closure
+ const authMiddleware = passport.authenticate('verify', function (error, user, infoOrError, status) {
+
+  if (error) {
+    return res.send(`TODO: render error-page with message ${error: error.message}`)
+  }
+
+  if (user) {
+    // passport-verify requires the use of a custom callback to handle successful
+    // authentication
+    return req.logIn(user, () => res.send('TODO: redirect to service landing page')))
+  }
+
+  return res.send(`TODO: redirect to authentication failed page with ${error: infoOrError}`)
+
+})
+authMiddleware(req, res, next)
+```
+</details>
+
```

Comment:
I think this section will read more clearly if you can add a line break or two after the folds.

Reply:
I agree!
But @jakubmiarka  and I had decided to remove example code blocks for anything that's for the legacy setup, since they won't be primary users. Still keeping the links to the API reference for everything, which is comprehensive enough.